### ISSUE TITLE
Why batch_size is setted to 4?

### ISSUE THREAD

**Issue by EdmunddzzZ** [muzairkhattak/multimodal-prompt-learning#16]
When train the model in a RTX 3090，found that the memory-usage was about 11G, so why not make it larger? 
The dataset is ImageNet

**Comment by muzairkhattak** [comment]
Hi @EdmunddzzZ,

Thank you for showing interest in MaPLe.

In our initial experiments, we found batch size of 4 to provide stable results and thus we fixed it. However, BS is an hyper-parameter and it can be further tuned for more optimal GPU utilization and performance. 

Feel free to adjust the BS according to your needs. Just make sure you also adjust/scale the learning rate accordingly.

Kindly let us know incase you have any other query. 

Thank you!

**Comment by EdmunddzzZ** [comment]
Thank you for patiently answering
